docs/source/multi_gpu.rst

```diff
@@ -654,7 +654,7 @@ To use Sharded Training, you need to first install FairScale using the command b
 
 .. code-block:: bash
 
-    pip install https://github.com/PyTorchLightning/fairscale/archive/pl_1.1.0.zip
+    pip install fairscale
```

Comment:
Shall we update it also in requirements?

Reply:
I made a note of it in the main post. We can but it isn't necessary, the unit test for the additional fixes is in Fairscale. Till, pipe RPC becomes a bit more stable let's keep this pointed to our own fork!